migrations/sql-annex/V2018.02.15.1500__RecreateBurdenEstimateStochasticWithSmallIntFields.sql

```diff
@@ -0,0 +1,24 @@
+DROP TABLE burden_estimate_stochastic;
+
+CREATE TABLE burden_estimate_stochastic (
+  id BIGSERIAL NOT NULL,
+  burden_estimate_set integer NOT NULL,
+  country text NOT NULL,
+  country_nid SMALLINT NULL,
+  year SMALLINT NOT NULL,
+  burden_outcome SMALLINT NOT NULL,
+  stochastic boolean NOT NULL,
```

Comment:
this column can be dropped (I thought it had been in a previous migration actually)

Reply:
ah you're right, I think I accidentally brought in back in here